index.html.liquid

```diff
@@ -478,7 +494,7 @@
                 <div class="row">
                   <div class="col-xl-12 col-sm-12">
 
-                    <a class="btn btn-success" {% if plan.system_name contains "partner" or plan.system_name contains "premium" %}mailto="{{ service.support_email }}"{% else %}href="{{ urls.signup }}?{{ plan | to_param }}"{% endif %}>Signup to {{ plan.name }}</a>
+                    <a class="btn btn-success" {% if plan.system_name contains "partner" or plan.system_name contains "premium" %}mailto="{{ provider.support_email }}"{% else %}href="{{ urls.signup }}?{{ plan | to_param }}"{% endif %}>Signup to {{ plan.name }}</a>

```

Comment:
any reason why it changed from `service` to `provider` @ManfredBo ?

Reply:
no idea. I didnt do that.

Reply:
okay I guess because the version of the file you created your branch from wasn't based on latest master, I'll fix any differences then

Reply:
@ManfredBo could you change it to `service.support_email` please?

Reply:
could you also change the line 452 from `{{ plan.system_name }}` to `{{ plan.name }}`

Reply:
the above change would then fix #25